{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--db6587fb-0c1e-4f89-ab09-8b570d2322f3",
    "created": "2024-08-13T16:09:07.807193Z",
    "modified": "2024-08-13T16:09:07.807197Z",
    "name": "No Title Available",
    "description": "Nginx NJS v0.7.4 was discovered to contain an out-of-bounds read via njs_scope_value at njs_scope.h.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2022-34029"
        }
    ],
    "references": [
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://github.com/nginx/njs/issues/506"
        }
    ]
}